docs/docs/protocol-specs/public-vm/gen/_instruction-set.mdx

```diff
@@ -1095,7 +1095,6 @@ context.machineState.pc = loc`}
 - **Details**: Target location is an immediate value (a constant in the bytecode).
 - **Bit-size**: 48
 
-[![](/img/protocol-specs/public-vm/bit-formats/INTERNALCALL.png)](/img/protocol-specs/public-vm/bit-formats/INTERNALCALL.png)
```

Comment:
accident?

Reply:
💀 apparantly happens if you run clean and build on the docs